controller/src/main/java/io/pravega/controller/server/rpc/auth/PravegaAuthManager.java

```diff
@@ -0,0 +1,30 @@
+/**
+ * Copyright (c) 2017 Dell Inc., or its subsidiaries. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ */
+package io.pravega.controller.server.rpc.auth;
+
+import io.grpc.ServerBuilder;
+import io.pravega.controller.server.rpc.grpc.GRPCServerConfig;
+import java.util.ServiceLoader;
+
+public class PravegaAuthManager {
+    private final GRPCServerConfig serverConfig;
+
+    public PravegaAuthManager(GRPCServerConfig serverConfig) {
+        this.serverConfig = serverConfig;
+    }
+
+    public void registerInterceptors(ServerBuilder<?> builder) {
+        ServiceLoader<PravegaAuthHandler> loader = ServiceLoader.load(PravegaAuthHandler.class);
```

Comment:
For my own understanding, is this able to pick up the `PravegaDefaultAuthHandler` impl because it's in the same jar?    (meaning I don't see any jar metadata as described here: https://docs.oracle.com/javase/tutorial/ext/basics/spi.html, which I guess 3rd party extension jars would have to provide in their jars )

For integration purposes, is this the jar file that one would need to import in order to implement that interface: 
 pravega-shared-controller-api-* ?

Reply:
Git did not add the metadata file last time. Thanks for catching it. Adding it now. For third party implementation, we need to add maven dependency for Pravega Controller.